diff --git a/node_modules/react-native-mmkv/react-native-mmkv.podspec b/node_modules/react-native-mmkv/react-native-mmkv.podspec
index be617c1..e6df80b 100644
--- a/node_modules/react-native-mmkv/react-native-mmkv.podspec
+++ b/node_modules/react-native-mmkv/react-native-mmkv.podspec
@@ -25,6 +25,7 @@ Pod::Spec.new do |s|
     "GCC_PREPROCESSOR_DEFINITIONS" => "$(inherited) FORCE_POSIX",
   }
   s.compiler_flags = '-x objective-c++'
+  s.libraries = "z"
 
   s.source_files = [
     # react-native-mmkv
